include/mbedtls/ssl.h

```diff
@@ -124,6 +126,10 @@
 
 #define MBEDTLS_SSL_MAX_HOST_NAME_LEN           255 /*!< Maximum host name defined in RFC 1035 */
 
+#define MBEDTLS_SSL_MAX_SIG_HASH_ALGORITHMS     65534 /*!< Maximum size of list in sig-hash algorithm ext., in RFC 5246    */
+#define MBEDTLS_SSL_MAX_SUPPORTED_ECC           65534 /*!< Maximum size of list in supported elliptic curve ext., RFC 4492 */
```

Comment:
No, the size of any _sensible_ list of curves or signature-hash-pairs is way below this limit, but nothing prevents you from adding lots of repitition, and we shouldn't fail in this case.